prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirToHl7Converter.kt

```diff
@@ -91,7 +92,7 @@ class FhirToHl7Converter(
     }
 
     override fun checkForEquality(converted: Message, expectedOutput: Message): Boolean {
-        return converted.encode() == expectedOutput.encode()
```

Comment:
Not necessarily related to the fix, but noticed that this wasn't using the extension method when debugging this